packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3912,10 +3918,10 @@ class Flex extends MultiChildRenderObjectWidget {
 /// warning box on the edge that is overflowing. If there is room on the outside
 /// of the row, the amount of overflow is printed in red lettering.
 ///
-/// {@tool sample}
-///
 /// #### Story time
 ///
+/// {@tool sample}
```

Comment:
Let's remove the `{@tool}` enclosings here. Since this is not a viable sample, we could visually support that by not wrapping it in a "sample" box.

Reply:
Done!